backend/app/usecase/url/urlcreator_test.go

```diff
@@ -6,6 +6,7 @@ import (
 	"testing"
 	"time"
 
+	"github.com/short-d/short/app/usecase/risk"
 	"github.com/short-d/short/app/usecase/service"
 
```

Comment:
```suggestion
```

Reply:
Please delete this line and run `./scripts/format` again and see whether it works.